eng/pipelines/pipeline-generation.yml

```diff
@@ -63,6 +64,7 @@ jobs:
           --variablegroup ${{ parameters.Release_Secrets_for_GitHubIO_Publishing }}
           --variablegroup ${{ parameters.Secrets_for_DevOps_Notification_Configuration_and_User_Resolution }}
           --variablegroup ${{ parameters.APIReview_AutoCreate_Configurations }}
+        TestOptions: '--variablegroup ${{ parameters.Secrets_for_Resource_Provisioner }}'
```

Comment:
FYI: @mitchdenny @praveenkuttappan @danieljurek @benbp this will start creating the test pipelines automatically for JS. The tools do seem to support this but I enabled the same thing in the JS Prepare-Pipelines setup today https://github.com/Azure/azure-sdk-for-js/pull/13145 and the test run https://dev.azure.com/azure-sdk/internal/_build/results?buildId=681866&view=logs&j=12f1170f-54f2-53f3-20dd-22fc7dff55f9&t=59a85588-b0ba-5043-24c4-d9e29d89c6f6 created a number of test pipelines. We should be on the look out for any duplicate pipelines or issues around this.

Reply:
Ah I hadn't seen that conditional. Will keep an eye on things.